pkg/scheduling/scheduling.go

```diff
@@ -0,0 +1,38 @@
+package scheduling
+
+import (
+	"strconv"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	ScheduleOtherPolicy   = "SCHED_OTHER"
```

Comment:
As I commented above, I think `SCHED_IDLE` and `SCHED_BATCH `should be taken into account as well. All of them must have the same priority (0).

Reply:
Priority based checks are considered.